app/src/main/java/com/ncnf/map/NCNFMarker.java

```diff
@@ -16,16 +16,18 @@
     private final String title;
     private final String snippet;
     private final List<Event> eventList;
+    private final boolean isEvent;
 
-    public NCNFMarker(float lat, float lon, String title, String snippet, List<Event> eventList){
-        this(new LatLng(lat, lon), title, snippet, eventList);
+    public NCNFMarker(float lat, float lon, String title, String snippet, List<Event> eventList, boolean isEvent){
+        this(new LatLng(lat, lon), title, snippet, eventList, isEvent);
     }
 
-    public NCNFMarker(LatLng position, String title, String snippet, List<Event> eventList){
+    public NCNFMarker(LatLng position, String title, String snippet, List<Event> eventList, boolean isEvent){
```

Comment:
I'm not sure why the marker needs a boolean to say whether it contains events, as it appear to posses only a List of event

Reply:
It is because markers will be used to represent both Events and Organizers so it becomes useful when accessing an event/organizer's page when clicking on the marker info window